g3proxy/docker/lua.alpine.Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM nickblah/lua:5.4-luarocks-alpine AS lua
```

Comment:
For the luarocks part, how about install `apk add luarocks-5.4` in the final stage, and then create a symbolic link from /usr/bin/luarocks-5.4 to /usr/bin/luarocks?

Reply:
And also lua5.4-socket, lua5.4-pgmoon (for posgresql),  lua5.4-sql-mysql, lua5.4-sql-postgres are packaged in alpine community channel. I'm more prefer to use apk to install lua modules.